aspnetcore/fundamentals/index.md

```diff
@@ -83,7 +83,7 @@ The code to create a host is in `Program.Main` and follows the [builder pattern]
 
 ::: moniker range="<= aspnetcore-2.2"
 
-ASP.NET Core 2.x uses Web Host (the `WebHost` class) for web apps. The framework provides `CreateDefaultBuilder` extension methods that set up a host with commonly used options, such as the following:
+ASP.NET Core 2.x uses Web Host (the `WebHost` class) for web apps. The framework provides `CreateDefaultBuilder` convenience methods that set up a host with commonly used options, such as the following:
```

Comment:
ASP.NET Core 2.x uses Web Host (the `WebHost` class) for web apps. The framework provides `CreateDefaultBuilder` ~convenience methods~ that set up a host with commonly used options, such as the following:

Reply:
ASP.NET Core 2.x uses Web Host (the `WebHost` class) for web apps. The framework provides `CreateDefaultBuilder` that sets up a host with commonly used options, such as the following: